cockatrice/src/tab_deck_storage.cpp

```diff
@@ -270,6 +270,8 @@ void TabDeckStorage::actNewFolder()
     QString folderName = QInputDialog::getText(this, tr("New folder"), tr("Name of new folder:"));
     if (folderName.isEmpty())
         return;
+    std::string folder = folderName.toStdString();
+    std::replace(folder.begin(), folder.end(), '/', '\\'); // Fix #2429
```

Comment:
You can remove the comment; include that info in the git commit. A good comment here might be
```
// '/' isn't a valid filename character on *nix, replace it to avoid problems storing on disk
```